js/runner.go

```diff
@@ -572,10 +572,6 @@ func unPromisify(v goja.Value) goja.Value {
 	return v
 }
```

Comment:
I think we could move unpromisify in the common package also

Reply:
We could ... but nothing else uses it - and I prefer to keep `common` and `util` packages smaller then bigger. 

Arguably a lot of changes will happen around the ESM changes and some other module work - if it turns out this is used a lot we can move it then.